File: app/src/main/java/com/just_me/task/audiolistener/data/models/TopArtists.java
package com.just_me.task.audiolistener.data.models;

import com.google.gson.annotations.SerializedName;
import com.just_me.task.audiolistener.data.models.Artist;

import java.util.ArrayList;

public class TopArtists {
    @SerializedName("artist")
    public ArrayList<Artist> artists;
    @SerializedName("@attr")
    public MetaData metadata;

    public static class MetaData {
        public String country;
        public int page;
        public int perPage;
        public int totalPages;
        public int total;
    }
}

File: app/src/main/java/com/just_me/just_we/lastfmclient/entity/TopArtists.kt
package com.just_me.just_we.lastfmclient.entity
import com.google.gson.annotations.SerializedName


data class TopArtists(
        @SerializedName("artist") val artists: List<Artist> = ArrayList(),
        @SerializedName("@attr") val attr: Attr = Attr())

data class Attr(
    @SerializedName("country") val country: String = "",
    @SerializedName("page") val page: String = "",
    @SerializedName("perPage") val perPage: String = "",
    @SerializedName("totalPages") val totalPages: String = "",
    @SerializedName("total") val total: String = ""
)